kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/MessagePublishContext.java

```diff
@@ -45,6 +45,12 @@
     private long sequenceId;
     private long highestSequenceId;
     private String producerName;
+    private boolean enableDeduplication;
+
+    @Override
```

Comment:
can you please add a javadoc here ?
this looks like a little hack, if Pulsar behaviour changes KOP won't work

Reply:
Yes, is a little hack, the better solution is to add an option on pulsar side, so we can use it for manual control deduplication. 

Do you have any suggestions?

Reply:
This solution works for me at least for current versions of Pulsar

Let's commit this patch